app/v1/controllers/AuthorisedController.scala

```diff
@@ -21,7 +21,7 @@ import play.api.mvc._
 import uk.gov.hmrc.auth.core.Enrolment
 import uk.gov.hmrc.auth.core.authorise.Predicate
 import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+import uk.gov.hmrc.play.bootstrap.controller.BackendController
```

Comment:
Can you please revert back to use previous import as this import is deprecated. Info can be seen [here](https://github.com/hmrc/bootstrap-play/blob/master/bootstrap-backend-play-26/src/main/scala/uk/gov/hmrc/play/bootstrap/package.scala#L37)